common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -195,10 +206,12 @@ object ContentWidths {
       wide =            Some(1920.px))
   }
 
-  def getWidthsFromContentElement(hinting: ContentHinting, relation: ContentRelation): WidthsByBreakpoint = {
+  def getWidthsFromContentElement(contentType: String, hinting: ContentHinting, relation: ContentRelation): WidthsByBreakpoint = {
     relation match {
       case MainMedia => hinting.mainContentWidths
-      case _ => hinting.bodyContentWidths }
+      case BodyMedia if hinting == ContentWidths.Inline && contentType == "LiveBlog" => LiveBlogMedia.Inline
```

Comment:
That's the case for other content types, they also have different breakpoints. The question still remains; if other content types don't require scattering `if` statements, why does live blog? The widths by breakpoint system should work in a consistent way.